apps/project/tests/test_apis.py

```diff
@@ -38,7 +38,7 @@
 
 
 class ProjectApiTest(TestCase):
-    fixtures = ['apps/ary/fixtures/ary_template_data.json']
+    fixtures = ['ary_template_data.json']
```

Comment:
Could you explain why this is needed?

Reply:
giving full path was causing error while running tests @thenav56 dai.

Reply:
Okay